gulpfile.js

```diff
@@ -353,43 +347,43 @@ gulp.task('start-simulators', function() {
   registrationServerSimulator.start();
 });
 
-gulp.task('stop-simulators', function() {
+gulp.task('stop-simulators', () => {
   foxboxSimulator.stop();
   registrationServerSimulator.stop();
 });
 
-gulp.task('run-unit-tests', ['compile-unit-tests'], function(cb) {
+gulp.task('run-unit-tests', ['compile-unit-tests'], (cb) => {
   const server = new (require('karma').Server)(
-    { configFile: `${__dirname}/karma.conf.js`}, cb
+    { configFile: `${__dirname}/karma.conf.js` }, cb
   );
 
   server.start();
 });
 
-gulp.task('run-test-integration', function() {
+gulp.task('run-test-integration', () => {
   return gulp.src(
     `${TESTS_ROOT}{common,integration}/**/*_test.js`, { read: false }
-    )
+  )
     .pipe(mocha());
 });
 
-gulp.task('doc', function() {
+gulp.task('doc', () => {
   runSequence(['clobber-doc'], ['generate-doc'], ['copy-doc-badge']);
 });
 
-gulp.task('test-integration', function(cb) {
+gulp.task('test-integration', (cb) => {
   runSequence('start-simulators', 'run-test-integration', () => {
     // Tear down whatever the result is
     runSequence('stop-simulators', cb);
   });
 });
 
-gulp.task('run-test-e2e', function() {
+gulp.task('run-test-e2e', () => {
   return gulp.src(`${TESTS_ROOT}{common,e2e}/**/*_test.js`, { read: false })
     .pipe(mocha());
 });
 
-gulp.task('test', ['clobber-tests', 'build-production'], function(cb) {
+gulp.task('test', ['clobber-tests', 'build-production'], (cb) => {
```

Comment:
__proposal:__ it's becoming hard to track new warnings in EsLint output because of these 30 warnings related to `Unexpected block statement surrounding arrow body`. I'd rather disable this rule at all... What do you think?